go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/gobwas/glob v0.2.3
 	github.com/gogo/protobuf v1.2.1 // indirect
 	github.com/golang-collections/collections v0.0.0-20130729185459-604e922904d3
+	github.com/google/btree v1.0.0 // indirect
```

Comment:
Any idea where this came from?

Reply:
Nope. It doesn't go away for some reason.